wiki/index.md

```diff
@@ -1,94 +1,27 @@
 ---
 title: Wiki
 layout: default
-author: Doug, Jackson, & Tony
+author: Ryan Stewart
 ---
-More coming soon...
-================================
 
-# Available Software
-Some software available through [Citrix](http://it.engineering.oregonstate.edu/citrix).
+# RTRP Wiki
 
-## Code Reference Resources
-* [Unix Shell](http://swcarpentry.github.io/shell-novice/)
-* [C++](http://www.learncpp.com/)
-* [Python3](https://docs.python.org/3/), [Python2](https://docs.python.org/2/)
-* [MATLAB](http://www.mathworks.com/help/matlab/)
-  - [FREE - Matlab’s Online Training Courses](https://matlabacademy.mathworks.com/)
-* [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) - what this website was written in ::
-* And most importantly.... [Google](https://www.google.com/)
-  -  Leads to StackExchange mostly. *Can* be a great reference and a cool place to ask the programming community questions.
+Welcome!
+This wiki page is for current or future student who are interested in Dr. Palmer's research group.
```

Comment:
This wiki page is for current or future ~~student~~ students

Reply:
Thanks! I've updated and added an example of what the interactive media will look like.